### ISSUE TITLE
Training Policy Advice?

### ISSUE THREAD

**Issue by crh19970307** [matterport/Mask_RCNN#55]
Hi, I want to train my own dataset consisting of 10 classes and 30000 images with segmentation annotation. Can you give me some advice on how many epoches for each training stage and how much time it may take to train on two TITAN X GPU? Thanks a lot!

**Comment by waleedka** [comment]
This is hard to guess. If you start from the COCO weights then I would estimate you might need to train for 20K steps (if each epoch is 1000 steps then 20 epochs) on the base learning rate, then half as many steps on LR/10. This is just a guess, though. Simply train on the base learning rate until the losses stop decreasing, and then divide the learning rate by 10 and continue training until the loss stops decreasing. 

By the way, get the latest release and COCO weights as they include useful improvements.
